src/bundler.js

```diff
@@ -41,7 +41,7 @@ const pathToPosix = p => p.split(sep).join(posix.sep);
  * @property {Record<string, string>} [aliases] module aliases
  * @property {boolean} [profile] Enable bundler performance profiling
  * @property {Record<string, string>} [env]
- * @property {import('rollup').Plugin[]} [plugins]
+ * @property {import('rollup').Plugin[]} plugins
```

Comment:
This I'm less than sure about, maybe some case I'm not seeing?

Right now [`normalize-options` ensures `plugins` is never undefined](https://github.com/preactjs/wmr/blob/master/src/lib/normalize-options.js#L17), so as far as I can tell there's no reason for it to be optional. Not a big deal to have a empty array to fallback to (as was the case before this PR) but I don't see a need for it currently.